modules/main-rules/pages/transport.adoc

```diff
@@ -1,39 +1,61 @@
-= Transport
+= Transportation
```

Comment:
This would've always been a better title I think, and it helps to disambiguate from the ability (and from the flyer mission, though we need to improve those titles and filenames later anyway).

I should change the filename to match too - but I will also need to find and update all inbound links to use the new filename, so I'll see what you think of the edits first and sort that out once we're settled.

Reply:
Looks good to me.

Reply:
Done, so I'll resolve this. For future reference though, the pull request basically sees the renamed file as 'new' in comparison with the master branch it is comparing to. That's logical of course, but it makes it harder to see what is truly different from master, and it messes with the conversations and suggestions...

We do need to be free to rename files to match titles, but it seems that we also need to be aware of these limitations so we don't get into a pickle with any complex conversations or suggestion that might be in progress. As it is in the case there were only a couple of points to round off.